include/tvm/runtime/threading_backend.h

```diff
@@ -44,6 +44,20 @@ class ThreadGroup {
     */
   void Join();
 
+  /*!
+   * \brief configure the CPU id affinity
+   *
+   * \param mode The preferred CPU type (1 = big, -1 = little).
```

Comment:
please use an enum for these `AFFINITY_MODE_(BIG|LITTLE)` to take advantage of static checking